### ISSUE TITLE
Failing to launch

### ISSUE THREAD

**Issue by Bowiemb** [MaibornWolff/dcos-zeppelin#8]
Hi, 

I'm installing zeppelin through the catalog of dcos. The zeppelin service keeps failing to deploy with the message `Failed to launch container: discarded; Abnormal executor termination: unknown container`. I've restarted, killed, and reinstalled a few times, and I seem to have plenty of resources (only 20-30% of cpu and memory used and only 5% of disk used.)

Any pointers in the right direction?

Thanks!

**Comment by swoehrl-mw** [comment]
Hi @Bowiemb,
the first place to look is the mesos slave log on the nodes where DC/OS tried to start zeppelin.
I do not think missing ressources is the problem as in this case DC/OS would not even try to start the service.

**Comment by Bowiemb** [comment]
@swoehrl-mw Yes, I figured out it's not just zeppelin... Any service that tries to pull from Docker is failing. Looks like it's a `docker pull` issue. `Failed to run docker -H ... Error: No such object: mesos-d2f333a8-fef2-48fb-8b99-28c52c327790`. I'm not sure this is the exact object for zeppelin, but it was the same message for anything that tried to pull from Docker... I saw documentation regarding pulling from private repositories, but this is the public Mesosphere repository... Not sure what's going wrong yet. Any pointers in the right direction?
